src/seedu/addressbook/data/person/ReadOnlyPerson.java

```diff
@@ -24,7 +24,8 @@
      * Returns true if both persons have the same identity fields (name and telephone).
      */
     default boolean isSamePerson(ReadOnlyPerson other) {
-        return other == this || (other != null && other.getName().equals(this.getName())
+        return other == this 
+                || (other != null && other.getName().equals(this.getName())
                 && other.getPhone().equals(this.getPhone()));
```

Comment:
This is more readable?

1. I put both branches of the OR condition in parenthesis, for symmetry.
2. I put the three conditions of the 2nd branch of the OR branch in three separate lines. Putting two of them in one line and the 3rd one in a separate line is a bit odd.
3. I aligned `&&`and `||` to be just inside the starting `(` of the condition they belong to.

```java
return (other == this) 
        || (other != null 
            && other.getName().equals(this.getName())
            && other.getPhone().equals(this.getPhone())));